api/models.go

```diff
@@ -318,6 +319,10 @@ type PayInvoiceRequest struct {
 	Amount *uint64 `json:"amount"`
 }
 
+type GenerateOfferRequest struct {
+	Description string `json:"description"`
+}
```

Comment:
For sure will be useful to be able to specify also the amount and if the invoice is reusable or not (not sure if the latter is encoded inside the protocol, tho, need to double check)

If the amount is unspecified so set it to nil

Reply:
We just wanted to keep this simple to only generate variable amount offers for now, so I left it to only set Description, will def add this in later stages 🙌